pagerduty/resource_pagerduty_event_orchestration_path_router.go

```diff
@@ -113,6 +135,54 @@ func resourcePagerDutyEventOrchestrationPathRouter() *schema.Resource {
 	}
 }
 
+func checkDynamicRoutingRule(context context.Context, diff *schema.ResourceDiff, i interface{}) error {
```

Comment:
**Annotation:** This function performs the following checks and returns a plan error if any of the checks are violated:

- A Router can have at most one Dynamic Routing rule
- Dynamic Routing rule must be the first rule of the first ("start") set
- A Dynamic Routing rule, if placed correctly as the first rule of the first set, cannot have conditions and the route_to action